README.md

```diff
@@ -1,95 +1,89 @@
+[![Combine XSD Schema v3.2](https://github.com/rudnerbjoern/iTop-schema/actions/workflows/combine-schema.v3.2.yml/badge.svg)](https://github.com/rudnerbjoern/iTop-schema/actions/workflows/combine-schema.v3.2.yml)
 
-This project provides an **XSD schema** to validate `datamodel.xxx.xml` files used in the [iTop CMDB software](https://www.combodo.com/itop).
+This project provides an **XSD schema** to validate `datamodel.my-module.xml`
+files used in the [iTop CMDB software](https://www.itophub.io).
 
-The goal is to catch **syntax errors** and **inconsistencies** in your iTop data models *before* importing them into iTop.
+The goal is to catch **syntax errors** and **inconsistencies** in your datamodel
+files *before* importing them into iTop.
 
-## Quickstart
+Keep in mind that this schema definition does not guarantee that your datamodel
+file is fully supported by iTop.
+Always use the [iTop toolkit](https://github.com/Combodo/itop-toolkit-community/releases)
+to fully verify your extension and ensure compatibility with iTop.
 
-Change the second line of your `datamodel.xml` to this:
+## Current status
 
-```xml
-<itop_design version="3.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="https://rudnerbjoern.github.io/iTop-schema/itop_design.xsd">
-```
-
-This definition will always point to the latest stable version of the schema for iTop (currently version 3.2).
-
-Your IDE will (probably) do the rest.
-
-If you are not yet using an IDE you might try [Visual Studio Code](https://code.visualstudio.com/) by Microsoft and the [XML extension](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-xml) by Red Hat.
-
-Keep in mind that this schema definition does not guarantee that your datamodel file is fully supported by iTop.
-The schema has many optional elements and attributes that might be mandatory in different use cases.
-
-Always use the [iTop toolkit](https://github.com/Combodo/itop-toolkit-community) to fully verify your extension and ensure compatibility with iTop.
-
-## Current Status
+Supported iTop design versions and URL to use to reference the schema:
 
-- Supported iTop design versions:
-  - [Version 3.2](https://www.itophub.io/wiki/page?id=latest:customization:xml_reference#version_32)
-- The project is in an **early stage**:
-  - The XSD is still **incomplete**.
-  - Many **errors and gaps** are expected.
-  - More advanced validations (e.g. for attribute types and relations) are not yet fully implemented.
+- [Latest](https://rudnerbjoern.github.io/iTop-schema/itop_design.xsd):
+  This will always point to the datamodel reference version from the latest
+  release (STS or LTS) of iTop.
+- [Version 3.2](https://rudnerbjoern.github.io/iTop-schema/3.2/itop_design.xsd):
+  This version is released with iTop 3.2 LTS in August 2024, see also the
+  [schema changes](https://www.itophub.io/wiki/page?id=latest:customization:xml_reference#version_32).
```

Comment:
Maybe you prefer to see the actual URL?
```suggestion
- `https://rudnerbjoern.github.io/iTop-schema/itop_design.xsd`:
  This will always point to the datamodel reference version from the latest
  release (STS or LTS) of iTop.
- `https://rudnerbjoern.github.io/iTop-schema/3.2/itop_design.xsd`:
  This version is released with iTop 3.2 LTS in August 2024, see also the
  [schema changes](https://www.itophub.io/wiki/page?id=latest:customization:xml_reference#version_32).
```

Let me know.